.github/workflows/arm64-build-checks.yml

```diff
@@ -2,6 +2,9 @@ name: arm64 build checks
 
 on: workflow_dispatch
 
+permissions:
+  contents: read # to fetch code (actions/checkout)
```

Comment:
Hello @sashashura!

Do you know can GHA install packages in this job using `apt-get` with this permission?

Reply:
Yes. The permissions are for the GITHUB_TOKEN only. It defines what GH API can be called.

Reply:
Thanks!